prow/config/config.go

```diff
@@ -1142,21 +1137,6 @@ func defaultPeriodics(periodics []Periodic, c *Config) error {
 func (c *Config) finalizeJobConfig() error {
 	if c.decorationRequested() {
 
-		if c.Plank.DefaultDecorationConfig != nil {
-			if len(c.Plank.DefaultDecorationConfigs) > 0 {
-				return errors.New("both default_decoration_config and default_decoration_configs are specified")
-			}
-
-			logrus.Warning("default_decoration_config will be deprecated on April 2020, and it will be replaced with default_decoration_configs['*'].")
-			c.Plank.DefaultDecorationConfigs = make(map[string]*prowapi.DecorationConfig)
-			c.Plank.DefaultDecorationConfigs["*"] = c.Plank.DefaultDecorationConfig
-		}
-
-		if len(c.Plank.DefaultDecorationConfigs) == 0 {
-			return errors.New("both default_decoration_config and default_decoration_configs['*'] are missing")
-
-		}
-
 		if _, ok := c.Plank.DefaultDecorationConfigs["*"]; !ok {
 			return errors.New("default_decoration_configs['*'] is missing")
 		}
```

Comment:
It looks like we only validate the config at the "*" key (below this line) instead of validating the config at all provided keys. That would be a good thing to fix here as well.

Reply:
Done